themes/default/content/what-is/what-is-hashicorp-vault.md

```diff
@@ -0,0 +1,87 @@
+
+- **Public Cloud**: Vault is commonly used in public cloud environments like AWS, Azure, and Google Cloud Platform. It can manage secrets and sensitive data for cloud services and applications, leveraging cloud-specific features like IAM roles for authentication and auto-scaling.
+- **Hybrid Cloud**: In hybrid cloud setups, where some resources are managed on-premises and others in the cloud, Vault can serve as a centralized secrets manager. It can bridge on-premises and cloud environments, ensuring consistent secrets management across both.
+- **Multi-Cloud**: Organizations using multiple cloud providers often adopt Vault to manage secrets uniformly across these environments. This approach avoids vendor lock-in and ensures that the secrets management strategy remains consistent, regardless of the underlying cloud provider.
+- **Containerized Environments**: In architectures using container orchestration tools like Kubernetes, Vault is used to manage secrets for containers. It can dynamically generate secrets and inject them into containers, enhancing security in microservices architectures.
+- **Serverless Architectures**: For serverless functions (like AWS Lambda, Azure Functions), Vault can manage API keys and database credentials, providing them securely to serverless functions when needed.
+- **CI/CD Pipelines**: Vault is often integrated into CI/CD pipelines to provide dynamic secrets for deployment automation tools, build servers, and testing environments, ensuring these processes are secure and auditable.
+- **Microservices Architecture**: In microservices environments, Vault can provide each service with unique credentials, reducing the risk of lateral movement in case of a compromise.
+- **Edge Computing**: In edge computing scenarios, Vault can manage secrets used by edge devices and local data processing applications, ensuring security in distributed environments.
+
+### Conclusion
+
+Vault's role in each of these architectures is to provide the foundation of secure storage, strict access control, and auditing for secrets and sensitive data. When combined with Pulumi ESC, the capabilities extend even further, ensuring strong security and efficient management across your entire cloud infrastructure and target execution environments.
+
+Pulumi ESC used in combination with Hashicorp Vault enables teams to aggregate secrets and configuration from many sources, manage hierarchical collections of configuration and secrets (“environments”), and consume those configuration and secrets from a variety of different infrastructure and application services. Pulumi ESC works hand-in-hand with Pulumi IaC to simplify configuration management, but also works independently from Pulumi IaC, as a solution for managing environments, secrets and configuration for any application or infrastructure project
```

Comment:
do we have a docs page about how to use Vault with ESC? we should link out to that.